scripts/fdbshow

```diff
@@ -95,7 +95,10 @@ class FdbShow(object):
             if not fdb:
                 continue
 
-            ent = self.db.get_all('ASIC_DB', s, blocking=True)
+            ent = self.db.get_all(self.db.ASIC_DB, s)
```

Comment:
Is it possible to keep blocking call and use try/except?

Reply:
@ganglyu what is the motivation? Even with the `try-catch` statement, sometimes it may freeze from half an minute to 1 minute. That may still affect the FDB test.